tests/DirectiveTest.php

```diff
@@ -20,6 +20,6 @@ public function it_renders_the_current_year()
     {
         View::addLocation(__DIR__.'/stubs/views');
 
-        $this->assertEquals(date('Y'), view('directive')->render());
+        $this->assertSame(date('Y'), view('directive')->render());
```

Comment:
Using the `assertSame` to replace `assertEquasl` and it can make assertion equals checking strict.